features/async-iterators.yml

```
@@ -0,0 +1,13 @@
+name: Async iterators and the for await..of loop
+description: The `for await .. of` loop handles asynchronous data when working with async iterables like Promises or Generators.
```

I feel like "handles" is a bit generic. How about these proposals:

```suggestion
description: The `for await .. of` statement loops over asynchronously iterable data like Promises or Generators.
```

```suggestion
description: Asynchronous iterator objects, such as those returned by promises or generator functions, are iterable with the `for await .. of` loop.
```